ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.ts

```diff
@@ -0,0 +1,482 @@
+import Conditions from '../../../../../resources/conditions';
+import NetRegexes from '../../../../../resources/netregexes';
+import { UnreachableCode } from '../../../../../resources/not_reached';
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
+import ZoneId from '../../../../../resources/zone_id';
+import { RaidbossData } from '../../../../../types/data';
+import { NetMatches } from '../../../../../types/net_matches';
+import { TriggerSet } from '../../../../../types/trigger';
+
+// TODO: Silkie specify which puff to get behind in first Slipper Soap
+// TODO: Silkie specify where to point puff's tether
+// TODO: Silkie call puff to go to for safety
+// TODO: Silkie call when to go to boss for safety
+// TODO: Silkie cleanup timeline
+// TODO: Gladiator triggers and timeline to enrage
+// TODO: Shadowcaster triggers and timeline to enrage
+
+export interface Data extends RaidbossData {
+  suds?: string;
+  soapCounter: number;
+  beaterCounter: number;
+  mightCasts: (NetMatches['StartsUsing'])[];
+  mightDir?: string;
+  hasLingering?: boolean;
+  thunderousEchoPlayer?: string;
+}
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.AnotherSildihnSubterrane,
+  timelineFile: 'another_sildihn_subterrane.txt',
+  initData: () => {
+    return {
+      soapCounter: 0,
+      beaterCounter: 0,
+      mightCasts: [],
+    };
+  },
+  triggers: [
+    // ---------------- Silkie ----------------
```

Comment:
```suggestion
    // ---------------- first trash ----------------
    {
      id: 'ASS Atropine Spore',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7960', '7978'], source: 'Aqueduct Belladonna', capture: false }),
      response: Responses.getIn(),
    },
    {
      id: 'ASS Frond Affront',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7961', '7979'], source: 'Aqueduct Belladonna', capture: false }),
      response: Responses.lookAway(),
    },
    {
      id: 'ASS Deracinator',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7962', '797A'], source: 'Aqueduct Belladonna', capture: false }),
      response: Responses.tankBuster(),
    },
    {
      id: 'ASS Left Sweep',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7964', '797C'], source: 'Aqueduct Kaluk', capture: false }),
      response: Responses.goRight(),
    },
    {
      id: 'ASS Right Sweep',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7963', '797B'], source: 'Aqueduct Kaluk', capture: false }),
      response: Responses.goLeft(),
    },
    {
      id: 'ASS Creeping Ivy',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7965', '797D'], source: 'Aqueduct Kaluk', capture: false }),
      response: Responses.getBehind(),
    },
    {
      id: 'ASS Honeyed Left',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['795B', '7973'], source: 'Aqueduct Udumbara', capture: false }),
      response: Responses.goRight(),
    },
    {
      id: 'ASS Honeyed Right',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['795C', '7974'], source: 'Aqueduct Udumbara', capture: false }),
      response: Responses.goLeft(),
    },
    {
      id: 'ASS Honeyed Front',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['795D', '7975'], source: 'Aqueduct Udumbara', capture: false }),
      response: Responses.getBehind(),
    },
    {
      id: 'ASS Arboreal Storm',
      type: 'StartsUsing',
      netRegex: NetRegexes.startsUsing({ id: ['7957', '796F'], source: 'Aqueduct Dryad', capture: false }),
      response: Responses.getOut(),
    },
    // ---------------- Silkie ----------------
```

Reply:
NB: this suggestion includes ASS and ASS Savage IDs

Reply:
I don't know if players are doing this, but does cactbot ever handle multiple mobs pulled at once that could cause multiple of these triggers to fire? Some of it I guess is just an if you know the encounter then you know which entities the call is for.

Reply:
The tankbuster response needs to have the capture. Could you remove the capture: false?

Reply:
Sorry about that, I'm grabbing portions from my js test triggers where it's not required.

Reply:
Thanks for the update.